prefect_kubernetes/jobs.py

```diff
@@ -299,3 +305,136 @@ def kubernetes_orchestrator():
             namespace=namespace,
             **kube_kwargs,
         )
+
+
+@flow(validate_parameters=False)
+async def run_namespaced_job(
+    kubernetes_credentials: KubernetesCredentials,
+    job_to_run: Union[V1Job, Dict[str, Any], Path, str],
```

Comment:
Would it make sense if this was a block (like a job block)?

Reply:
Also, would it make sense to accept a KubernetesManifest?

Reply:
yes I think it would make sense for this to accept some `JobBlock`, asking alex about this

`KubernetesManifest` is just a type wrapper for a `Dict[str, str]`, so I don't feel that's necessary before moving that stuff out of core